### PR TITLE
Feature/show all metrics and table cells

### LINKED ISSUE #773: [FE] Default experiment tracking view should be show all metrics
## Description

The default experiment tracking view should be show all metrics instead of only showing common issues, which is what we currently do.

## Checklist

- [x] Include labels so that we can categorise your feature request

### PR DESCRIPTION
**Pull request by tynandebold**
## Description

Resolves #773 and #780.

## Development notes

First, I set `showDiff` to true in the `experiment-wrapper.js` file.

Next, the real crux of this is the `fillEmptyMetrics()` function in the `run-dataset.js` file. This is what fills in the empty data, which results in the correct empty cells in the table now being displayed.

I also added some comments to the code and expanded on some that were already there, as they were lacking some arguments.

I found some missing styles on the `focus` states on the buttons, too, so I've added those in.

There are some smaller changes here as well, mostly resulting from auto formatting the readme and release notes file.

Lastly, I was seeing this console warning:

```bash
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
```

so I ran that command, and that's why there are changes in the `package.json` and `package-lock.json` files.

## QA notes

To test this, select the run with the name `2021-12-16T09.45.55.269Z`. That has a metric called `model_author` with the value of `richard feynman`. That differs from any of the other runs. If you select this run along with any of the others, you should be able to see that this metric is shown and the other runs that don't have this metric have the `-` in the table cell(s).

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [x] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/798"></a>

### CODE REVIEW

**Review by rashidakanchwala — APPROVED** [review]
Really nice implementation @tynandebold ! :)

File: src/components/experiment-tracking/run-dataset/run-dataset.js [hunk 1]
```diff
@@ -61,15 +65,18 @@ const RunDataset = ({
             size="large"
           >
             {Object.keys(data)
-              .sort()
+              .sort((a, b) => {
+                return a.localeCompare(b);
+              })
```
Comment (studioswong) [inline src/components/experiment-tracking/run-dataset/run-dataset.js:70]: Out of curiosity, I wonder if there is any specific reasons why you prefer to use `localeCompare` other than the normal `sort` method?

**Review by studioswong — APPROVED** [review]
I've tested this and it works well, so approving straight away.

One suggestion - it would be nice to add a test to test the expected behaviour for pre-filling / displaying the empty value as a `-` - perhaps something to test the `fillEmptyMetrics` function under `run-dataset.test.js`?

File: README.md [hunk 2]
```diff
@@ -39,26 +39,26 @@ Kedro-Viz is an interactive development tool for building data science pipelines
...
+  pip install kedro-viz
+  ```
 
-    ```bash
-    pip install kedro-viz
-    ```
+- As a standalone React component (for embedding Kedro-Viz in your web application).
 
-* As a standalone React component (for embedding Kedro-Viz in your web application).
+  To install the standalone React component:
```
[the remark this answers is not in the document]
Comment (tynandebold) [reply README.md:56]: Agreed. Somehow this line was moved to the line above, so it's still in there and does hopefully give the context for the users.

The lines around this one you've commented on look like this:

```md
- As a standalone React component (for embedding Kedro-Viz in your web application).

  To install the standalone React component:

  npm install @quantumblack/kedro-viz
```

**Comment by tynandebold** [comment]
> I've tested this and it works well, so approving straight away.
> 
> One suggestion - it would be nice to add a test to test the expected behaviour for pre-filling / displaying the empty value as a `-` - perhaps something to test the `fillEmptyMetrics` function under `run-dataset.test.js`?

Great idea about the test. I've added one [here](https://github.com/kedro-org/kedro-viz/pull/798/commits/617e6387f4166f4770d6feb980bedda4750a446a#diff-f83a82ba63de532116ba34aefa80a2f2ec7fb7834801c64bebb1ba78003f311eR102-R113) :)
